### PR TITLE
Concf 1046 fix empty image url when image id points to non-file page

### PR DESCRIPTION
**Pull request by bognix**
/cc @hakubo @vforge

### CODE REVIEW

**Comment by Warkot** [comment]
LGTM

**Comment by vforge** [comment]
LGTM

File: extensions/wikia/CuratedContent/CuratedContentController.class.php [hunk 1]
```diff
@@ -549,13 +549,14 @@ private function getCuratedContentForWiki( $wikiID ) {
...
 		$return = [ ];
-		foreach ( $sections as $section ) {
-			$categoriesForSection = $this->getSectionItems( $content, $section['title'] );
-			foreach ( $categoriesForSection as $category ) {
-				$return[] = $category;
+		if( is_array( $sections ) ) {
+			foreach ( $sections as $section ) {
+				$categoriesForSection = $this->getSectionItems( $content, $section['title'] );
+				foreach ( $categoriesForSection as $category ) {
+					$return[] = $category;
```
Comment (hakubo) [inline extensions/wikia/CuratedContent/CuratedContentController.class.php]: can we rename return to what it actually is?

File: extensions/wikia/CuratedContent/tests/CuratedContentHelperTest.php [hunk 2]
```diff
@@ -0,0 +1,71 @@
+<?php
```
Comment (hakubo) [inline extensions/wikia/CuratedContent/tests/CuratedContentHelperTest.php:1]: :*

**Comment by hakubo** [comment]
LGTM

**Comment by tomasznapieralski** [comment]
LGTM

**Pull request merged**